src/client/java/daos-java-api/src/test/native/test.c

```diff
@@ -0,0 +1,141 @@
+/*
+ * test.c
+ *
+ *  Created on: Jan 9, 2020
+ *      Author: jiafu
+ */
+#include <gurt/common.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/xattr.h>
+#include <dirent.h>
+
+#include <libgen.h>
+#include <stdio.h>
+#include <daos.h>
+#include <daos_fs.h>
+#include <fcntl.h>
+#include <daos_obj_class.h>
+
+int main(int argc, char *argv[]) {
+	if (argc != 6) {
+		printf("need arguments of server group, pool UUID, container UUID, file name and file length\n");
+		return 1;
+	}
+	char *server_group = argv[1];
+	char *pool_str = argv[2];
+	char *cont_str = argv[3];
+	char *file_name = argv[4];
+	int file_len = atoi(argv[5]);
+	int rc = daos_init();
+	if (rc) {
+		printf("daos_init() failed with rc = %d\n", rc);
+		return rc;
+	}
+	//connect to pool
+	uuid_t pool_uuid;
+	uuid_parse(pool_str, pool_uuid);
+	d_rank_list_t *svcl = daos_rank_list_parse("0", ":");
+	daos_handle_t poh = {0};
+	rc = daos_pool_connect(pool_uuid, server_group, svcl,
+					2, /* read write */
+					&poh /* returned pool handle */,
+					NULL /* returned pool info */,
+					NULL /* event */);
+
+	if (rc) {
+		printf("Failed to connect to pool (%s), rc is %d\n", pool_str, rc);
+		return rc;
+	}
+	//connect to container
+	uuid_t cont_uuid;
+	uuid_parse(cont_str, cont_uuid);
+	daos_cont_info_t co_info;
+	daos_handle_t coh = {0};
+	rc = daos_cont_open(poh, cont_uuid, 2, &coh, &co_info, NULL);
+	if (rc) {
+		printf("Failed to connect to container (%s)\n", cont_str);
+		goto quit;
+	}
+	//mount FS
+	dfs_t *dfs = NULL;
+	rc = dfs_mount(poh, coh, O_RDWR, &dfs);
+	if(rc){
```

Comment:
(style)  space required before the open brace '{'
(style)  space required before the open parenthesis '('